frontend/packages/ceph-storage-plugin/integration-tests/utils/helpers.ts

```diff
@@ -159,6 +159,15 @@ export const verifyNodeOSDMapping = (
   return filteredOsds.length === 0;
 };
 
+export const hasTaints = (node) => {
+  return !_.isEmpty(node.spec?.taints);
+};
+
+export const hasOCSTaint = (node) => {
+  const taints = node?.spec?.taints || [];
+  return taints.some((taint) => _.isEqual(taint, ocsTaint));
+};
```

Comment:
Why you have defined the functions again here? You can reuse the function from `node-list.tsx`

Reply:
You can't pull that code into e2e tests and vice versa,